File: AlphaStandardDemo/Models/Carro.vb
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports AlphaStandardDemo.Interfaces

Namespace Models
  <Table("carros", Schema:="cadastros")>
  Public Class Carro
    Implements IEntity

    <Key>
    Public Property Id As Integer Implements IEntity.Id
    Public Property Marca As String
    Public Property Modelo As String
    Public Property Cor As String
    Public Property Ano As Integer

    Public Overrides Function ToString() As String
      Return $"{Marca} - {Modelo} - {Ano} - {Cor}"
    End Function
  End Class
End Namespace

File: AlphaLib/Models/Carro.cs
using AlphaLib.Interfaces;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AlphaLib.Models
{
    [Table("carros", Schema = "cadastros")]
    public class Carro : IEntity
    {
        [Key]
        public int Id { get; set; }
        public string Marca { get; set; }
        public string Modelo { get; set; }
        public string Cor { get; set; }
        public int Ano { get; set; }

        public override string ToString() => $"{Marca} - {Modelo} - {Ano} - {Cor}";
    }
}
